src/core/Spec.js

```diff
@@ -57,7 +57,8 @@ getJasmineRequireObj().Spec = function(j$) {
       passedExpectations: [],
       deprecationWarnings: [],
       pendingReason: '',
-      duration: null
+      duration: null,
```

Comment:
Could you please also update the jsdoc comment just above this?

Reply:
done